package-lock.json

```diff
@@ -0,0 +1,6388 @@
+{
```

Comment:
Delete this file. We use `yarn` as our default package manager. `package-lock.json` is generated because you are using `npm`.